homeassistant/components/light/strings.json

```diff
@@ -86,5 +86,307 @@
+        "orange": "Orange",
+        "orangered": "Orange red",
+        "orchid": "Orchid",
+        "palegoldenrod": "Pale goldenrod",
+        "palegreen": "Pale green",
+        "paleturquoise": "Pale turquoise",
+        "palevioletred": "Pale violet red",
+        "papayawhip": "Papaya whip",
+        "peachpuff": "Peach puff",
+        "peru": "Peru",
+        "pink": "Pink",
+        "plum": "Plum",
+        "powderblue": "Powder blue",
+        "purple": "Purple",
+        "red": "Red",
+        "rosybrown": "Rosy brown",
+        "royalblue": "Royal blue",
+        "saddlebrown": "Saddle brown",
+        "salmon": "Salmon",
+        "sandybrown": "Sandy brown",
+        "seagreen": "Sea green",
+        "seashell": "Seashell",
+        "sienna": "Sienna",
+        "silver": "Silver",
+        "skyblue": "Sky blue",
+        "slateblue": "Slate blue",
+        "slategray": "Slate gray",
+        "slategrey": "Slate grey",
+        "snow": "Snow",
+        "springgreen": "Spring green",
+        "steelblue": "Steel blue",
+        "tan": "Tan",
+        "teal": "Teal",
+        "thistle": "Thistle",
+        "tomato": "Tomato",
+        "turquoise": "Turquoise",
+        "violet": "Violet",
+        "wheat": "Wheat",
+        "white": "White",
+        "whitesmoke": "White smoke",
+        "yellow": "Yellow",
+        "yellowgreen": "Yellow green"
+      }
+    }
+  },
+  "services": {
+    "turn_on": {
+      "name": "Turn on",
+      "description": "Turn on one or more lights and adjust properties of the light, even when they are turned on already.",
+      "fields": {
+        "transition": {
+          "name": "Transition",
+          "description": "Duration it takes to get to next state."
+        },
+        "rgb_color": {
+          "name": "Color",
+          "description": "The color for the light (based on RGB - red, green, blue)."
+        },
+        "rgbw_color": {
+          "name": "RGBW-color",
+          "description": "A list containing four integers between 0 and 255 representing the RGBW (red, green, blue, white) color for the light."
+        },
+        "rgbww_color": {
+          "name": "RGBWW-color",
+          "description": "A list containing five integers between 0 and 255 representing the RGBWW (red, green, blue, cold white, warm white) color for the light."
+        },
+        "color_name": {
+          "name": "Color name",
+          "description": "A human readable color name."
+        },
+        "hs_color": {
+          "name": "Hue/Sat color",
+          "description": "Color for the light in hue/sat format. Hue is 0-360 and Sat is 0-100."
+        },
+        "xy_color": {
+          "name": "XY-color",
+          "description": "Color for the light in XY-format."
+        },
+        "color_temp": {
+          "name": "Color temperature",
+          "description": "Color temperature for the light in mireds."
+        },
+        "kelvin": {
+          "name": "Color temperature (Kelvin)",
```

Comment:
For a future PR, I wonder if things like `(Kelvin)` are necessary since we could just show the unit of measurement behind the value.

Reply:
I honestly think we can remove it here, as it is part of the description and the selector shown already.

Reply:
```suggestion
          "name": "Color temperature",
```